.github/scripts/check-ga-release-deps.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+set -ue
+ 
+release_title=$(echo "$RELEASE_TITLE" | tr '[:upper:]' '[:lower:]')
+badfiles="false"
+
+if echo "$release_title" | grep -Eq '^(release v([0-9]{1,2}|100)\.[0-9]{1,100}\.[0-9]{1,100}|v([0-9]{1,2}|100)\.[0-9]{1,100}\.[0-9]{1,100})$'; then
+    for FILE in Dockerfile.dapper go.mod ./package/Dockerfile ./pkg/apis/go.mod ./pkg/settings/setting.go ./scripts/package-env; do
```

Comment:
Let's create a variable to hold these files. Will make adding or removing easier than editing the processing code itself.